core/src/scss/elements/input/_input.scss

```diff
@@ -8,7 +8,8 @@
 input,
 textarea,
 select {
-  appearance: none;
+  -moz-appearance: none;
```

Comment:
We shouldn't need vendor prefixes as the autoprefixer should take care of this for us.

Reply:
Oh @sherakama , this is the weird one, the one without prefix is invalid, so the only forms that are actually valid are the prefix ones so I thought I'd remove the un-prefixed one.

Reply:
This comment from https://developer.mozilla.org/en-US/docs/Web/CSS/appearance concerns me:

> **Compatibility note**: If you wish to use this property on websites, you should test it very carefully. Although it is supported in most modern browsers, its implementation varies widely. In older browsers, even the keyword none does not have the same effect on all form elements across different browsers, and some do not support it at all. The differences are smaller in the newest browsers.

Should we be using `appearance` at all?

Reply:
Actually, yeah, @JBCSU , I'm not really sure we need it for the browsers that we're supporting (2 versions back). @sherakama , what do you think? Could we get rid of appearance?

Reply:
+1 to removing it.

Reply:
+1 also. Please remove it.

Reply:
Gladly 😄

Reply:
Ok, it's gone from both input and button-base